Adds checkboxes for code formatting and testing.

ryankoul opened the pull request:
## Description ##

Clearly documents the reason for the pull request and what has changed.
- [ ] This is a bugfix.
- [ ] This is a new feature.
- [ ] This is part of a feature.

## Organization ##

- [ ] Have you linked this pull request to a Trello card?

## Review ##

- [ ] Have you added two reviewers to this pull request?
- [ ] Is this a work in process?
- [ ] Is this ready to be reviewed?

ryankoul committed:
Adds checkboxes for code formatting and testing.

nusc2016 commented:
Looks good

Bhavani-Rajan reviewed (changes requested):
Instead format code, can you rename it as api endpoints?

ryankoul committed:
Updates pull_request_template

Specifies how code will be formatted (autopep-8) and how code will be tested (unit tests).

nusc2016 reviewed (approved):
Looks good to me
